ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -242,6 +244,11 @@ let options =
     , (fun () -> !pvinpvh_xen_cmdline)
     , "Command line for the inner-xen for PV-in-PVH guests"
     )
+  ; ( "pvh-ovmf-cmdline"
+    , Arg.Set_string pvh_ovmf_cmdline
+    , (fun () -> !pvh_ovmf_cmdline)
+    , "Command line for OVMF for PVH guests"
+    )
```

Comment:
This is is probably ok for a prototype, but we're going to need to improve the data model in due course

Reply:
This is currently completely unused, so I might as well drop it. I think for pvinpvh this is needed because there is also a nested Xen and you may want to control the args to that independently of the args for the guest kernel, but for PVH the args to the guest kernel are controllable directly and there is no nested Xen.